lib/msf/core/exploit/remote/kerberos/auth_brute.rb

```diff
@@ -99,6 +111,12 @@ def attempt_kerberos_logins
         else
           vprint_status("#{peer} - User: #{user.inspect} - #{proof}")
         end
+      when Metasploit::Model::Login::Status::LOCKED_OUT
+        print_error("#{peer} - User: #{user.inspect} account locked out")
+      when Metasploit::Model::Login::Status::DISABLED
+        print_error("#{peer} - User: #{user.inspect} account disabled or expired")
+      when Metasploit::Model::Login::Status::DENIED_ACCESS
+        print_error("#{peer} - User: #{user.inspect} account unable to log in")
       end
```

Comment:
Should we future proof this to log out the proof in the default case?

```suggestion
      else
          print_error("#{peer} - User: #{user.inspect} #{proof}")
      end
```